src/domain/collection/CollectionPageContainer.tsx

```diff
@@ -35,50 +39,78 @@ const CollectionPageContainer: React.FC = () => {
   });
   const collection = collectionData && collectionData.collectionDetails;
 
+  const getErrorHero = (collection?: CollectionFieldsFragment) => {
+    if (!collection) {
+      return (
+        <ErrorHero
```

Comment:
maybe add renderErrorHero helper function that takes text, title, to and linkTexts as parameters to avoid <ErrorHero /> / <Link /> repetition.

Just a suggestion, not sure if it's any better :D

Reply:
Refactored way to get errors little bit. I hope readability is better now